frontend/src/pages/audits/list/audits-list.html

```diff
@@ -123,7 +123,7 @@
                 </q-chip>
             </q-td>
             <q-td slot="body-cell-action" slot-scope="props" style="width:1px">
-                <audit-state-icon size="sm" :approvals="props.row.approvals" :state="props.row.state" :minReviewers="settings.minReviewers" class="icon-next-to-button"/>
+                <audit-state-icon size="sm" :approvals="props.row.approvals" :state="props.row.state" class="icon-next-to-button"/>
```

Comment:
I don't think those should be visible when the review process is disabled. Are you maybe missing a v-if?